anomalib/models/dfm/config.yaml

```diff
@@ -14,6 +14,8 @@ dataset:
 model:
   name: dfm
   backbone: resnet18
+  layer: layer3
+  pool: 4
```

Comment:
Maybe this could be renamed to `pool_factor` to make the parameter a bit more self explanatory